### PR TITLE
1.4 release and kubecon blog for demo

### PR DESCRIPTION
**Pull request by cartersocha**

### CODE REVIEW

File: content/en/blog/2023/demo-birthday.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: The OpenTelemetry Demo Turns 1(.4)
+linkTitle: OTel Demo Updates
+date: 2023-04-16
+author: '[Carter Socha](https://github.com/cartersocha) (Lightstep)'
+---
+
+It's hard to belive as we prepare our 1.4.0 release but the [OpenTelemetry
```
Comment (fatsheep9146) [inline content/en/blog/2023/demo-birthday.md:8]: typo believe

**Review by julianocosta89 — APPROVED** [review]
Content wise it LGTM 🤩

**Review by fatsheep9146 — APPROVED** [review]

**Review by svrnm — APPROVED** [review]
happy to have that, looks good to me content-wise, please make sure that you fix the formatting

**Comment by cartersocha** [comment]
I'm getting a really annoying error locally trying to fix the check links / refcache. What am I missing? Are there version requirements? @chalin @svrnm

**Comment by cartersocha** [comment]
❗ Planning on adding a header picture. Please don't merge until it's added ❗

File: content/en/blog/2023/demo-birthday.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+- _Support added for arm64 architectures (M1 and M2 Macs)_
+- **Async support using Kafka and 2 new services**
+- _Kubernetes manifest_
+- **More out of the box dashboards**
+- _Our first logging SDK addition_
+- **A myriad of frontend bug fixes**
+- _First
+  [Connector](https://github.com/open-telemetry/opentelemetry-collector/blob/main/connector/README.md)
+  in the Collector_
+- **New metric SDKs**
```
Comment (avillela) [inline content/en/blog/2023/demo-birthday.md:36]: Such as?

File: content/en/blog/2023/demo-birthday.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+- _New manual metric instruments_
+- **More auto-instrumentation in more languages**
```
Comment (avillela) [inline content/en/blog/2023/demo-birthday.md:38]: Would be interesting to say how many languages we started out with, and how many we have now

File: content/en/blog/2023/demo-birthday.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+Time flies when you're stabilizing semantic conventions. But what's actually
+changed between our 1.0.0 and 1.4.0 releases? Quite a lot actually.
+
+### The Highlights
+
+- **2x build time improvements despite adding additional services**
+- _Support added for arm64 architectures (M1 and M2 Macs)_
+- **Async support using Kafka and 2 new services**
+- _Kubernetes manifest_
```
Comment (avillela) [inline content/en/blog/2023/demo-birthday.md:29]: I feel like this requires a bit more explanation? I'm guessing that this means that you can deploy to k8s via k8s manifest or Helm?

File: content/en/blog/2023/demo-birthday.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+- **More out of the box dashboards**
+- _Our first logging SDK addition_
```
Comment (avillela) [inline content/en/blog/2023/demo-birthday.md:31]: Link to Logging SDK docs

File: content/en/blog/2023/demo-birthday.md (hunk 3 above)
Comment (avillela) [inline content/en/blog/2023/demo-birthday.md:38]: Some examples of where we started, and where we're at would be helpful

**Comment by cartersocha** [comment]
It wasn’t ready to merge @austinlparker

**Comment by austinlparker** [comment]
Whoops! It was approved and all comments were resolved? Can you make a new PR with whatever changes were left?Sent from my iPhoneOn Apr 18, 2023, at 7:39 AM, Carter Socha ***@***.***> wrote:﻿
It wasn’t ready to merge @austinlparker

**Comment by austinlparker** [comment]
Ok, I missed the comment, my bad. In my defense it's early.

**Comment by cartersocha** [comment]
All good. I’ll just update later and then point to it

**Comment by austinlparker** [comment]
You'll need to create a new PR -- sorry, I saw your message in Slack to revert and did so.

**Pull request merged**